src/const.ts

```diff
@@ -62,6 +62,7 @@ export enum NodeType {
     Device = 'ha-device',
     Entity = 'ha-entity',
     EventsAll = 'server-events',
+    EventsCalendar = 'events-calendar',
```

Comment:
prefix the node type with `ha-`
```suggestion
    EventsCalendar = 'ha-events-calendar',
```